test/ClientTest.php

```diff
@@ -11,8 +11,8 @@ public function testHashFunctions()
         $values = [
             'one'   => 'abc',
             'two'   => 'def',
-            'three' => 123,
-            'four'  => 1.0 / 3
+            'three' => '123',
```

Comment:
I don't recally details about this anymore, but to me this test looks like it would specifically check for inconsistent type array - is there a reason to change to to a string only array?

Reply:
Redis hash calues are string, so type-safe equality isn't given - therefore values will be strings when pulled from redis again.